pkg/users/accounts-list.js

```diff
@@ -18,72 +18,277 @@
+            const sortedRows = rows.sort((a, b) => {
+                const aitem = a.columns[idx];
+                const bitem = b.columns[idx];
+                const aname = a.columns[0];
+                const bname = b.columns[0];
+
+                // current user is always first
+                if (aname.sortKey === this.props.current_user)
+                    return direction === SortByDirection.asc ? -1 : 1;
+                if (bname.sortKey === this.props.current_user)
+                    return direction === SortByDirection.asc ? 1 : -1;
+                // sorting last login
+                if (idx === 3) {
+                    if (aitem.sortKey === "logged in")
+                        return -1;
+                    if (bitem.sortKey === "logged in")
+                        return 1;
+                    if (aitem.sortKey === "never")
+                        return 1;
+                    if (bitem.sortKey === "never")
+                        return -1;
+
+                    return bitem.sortKey - aitem.sortKey;
+                }
+
+                return ((typeof aitem == 'string' ? aitem : (aitem.sortKey || aitem.title)).localeCompare(typeof bitem == 'string' ? bitem : (bitem.sortKey || bitem.title)));
+            });
+            return direction === SortByDirection.asc ? sortedRows : sortedRows.reverse();
+        };
+
+        const tableToolbar = (
+            <Toolbar>
+                <ToolbarContent>
+                    <ToolbarItem spacer={{ default: 'spacerLg' }}>
+                        <Text component='h2'>Accounts</Text>
+                    </ToolbarItem>
+                    <ToolbarItem variant="search-filter">
+                        <SearchInput id="accounts-filter"
+                                     className="accounts-filter"
+                                     placeholder={_("Search for name, group or ID")}
+                                     value={this.state.currentTextFilter}
+                                     onChange={this.setCurrentTextFilter}
+                                     onClear={() => this.setCurrentTextFilter('')} />
+                    </ToolbarItem>
+                    { superuser.allowed &&
+                        <ToolbarItem alignment={{ default: 'alignRight' }}>
+                            <Button id="accounts-create" onClick={() => account_create_dialog(this.props.accounts)}>
+                                {_("Create new account")}
+                            </Button>
+                        </ToolbarItem>
+                    }
+                </ToolbarContent>
+            </Toolbar>
+        );
+
+        return (
+            <Page id="accounts">
+                <PageSection>
+                    <Card>
+                        {tableToolbar}
+                        {filtered_accounts.length === 0
+                            ? <EmptyStatePanel title={_("No matching results")} icon={SearchIcon} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17830-20221116-150219-482f17ff-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)